src/openforms/formio/components/custom.py

```diff
@@ -382,6 +383,13 @@ class AddressNL(BasePlugin):
 
     formatter = AddressNLFormatter
 
+    def build_serializer_field(self, component: Component) -> serializers.DictField:
+        validate = component.get("validate", {})
+        required = validate.get("required", False)
+        return serializers.DictField(
+            child=AddressValueSerializer(), required=required, allow_null=not required
+        )
```

Comment:
I'm not sure why tests passes currently, but `child` is used to validate the dict values, so what is being returned here actually validates this kind of data:

```json
{
    "myAdressNL": {
        "any_key": {"postcode": "..."}
    }
}
```

While we want to validate:

```json
{
    "myAdressNL": {
        "postcode": "..."
    }
}
```